src/components/pages/Dashboard/Dashboard.jsx

```diff
@@ -1,23 +1,30 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import "./_style.scss";
-import { Select, Button, Icon } from "antd";
-import { useState } from "react";
+import { Button, Icon } from "antd";
 import CustomDatePicker from "../../atoms/CustomDatePicker/CustomDatePicker";
 import CustomDropdown from "../../atoms/CustomDropdown/CustomDropdown";
-import { useEffect } from "react";
+import apiCalls from "../../../services/api-calls/all";
+
+import RingCharts from "../../molecules/RingCharts/RingCharts";
+
+const { getRingChartValues } = apiCalls();
 
 const Dashboard = () => {
   const [operators, setOperators] = useState();
   const [dateFrom, setDateFrom] = useState();
   const [dateTo, setDateTo] = useState();
-  const [operatorValue, setOperatorValue] = useState();
+  const [operatorValue, setOperatorValue] = useState(1);
+  const [ringChartValues, setRingChartValues] = useState();
 
   useEffect(() => {
     // TODO: read operators and setOperators
   }, []);
 
-  const handleSearch = () => {
-    // TODO: call functions necessary to get charts and tables
+  const handleSearch = async () => {
+    const ringsQuery = { dateFrom, dateTo };
```

Comment:
2 cosas: 
1- no me funciona si no pongo dateFrom y dateTo
2- deberiamos poner un default al iniciar el dashboard, por ej: traer lo de hoy (llenamos los filtros tmb)

Reply:
Si, sé, por eso agregué este TODO: 
`{/* TODO: make this button disabled if no dates are picked, and create a theme for that */}`
despues agrego estos cambios(y lo de operators) a un mr  q se llame `fixes dashboard filters `